src/metabase/driver/postgres.clj

```diff
@@ -435,6 +445,23 @@
    (keyword "timestamp with timezone")    :type/DateTime
    (keyword "timestamp without timezone") :type/DateTime})
 
+(doseq [[base-type db-type] {:type/BigInteger          "BIGINT"
+                             :type/Boolean             "BOOL"
+                             :type/Date                "DATE"
+                             :type/DateTime            "TIMESTAMP"
+                             :type/DateTimeWithTZ      "TIMESTAMP WITH TIME ZONE"
+                             :type/DateTimeWithLocalTZ "TIMESTAMPTZ"
```

Comment:
[`timestamp with time zone` and `timestamptz` are the same thing -- the latter is just an abbreviation for the former](https://www.postgresql.org/docs/9.1/datatype-datetime.html) -- so it's a little worrying that we're mapping different MB types to different names for the same thing

Reply:
If it helps `timestamptz` is basically a `:type/DateTimeWithLocalTZ` since according to the definition

> a column that is timezone-aware, but normalized to UTC or another offset at rest.

That's exactly what it is. That's the only time zone aware `timestamp` type in Postgres. So you probably want to map all `:type/DateTimeWithTZ` to `timestamp with time zone`

Reply:
This came up because I persisted a table locally and got an error. I checked the type of the column it was trying to persist and this is what it reported so i added it. I've updated now.